pyglotaran_extras/plotting/plot_data.py

```diff
@@ -41,6 +41,7 @@ def plot_data_overview(
     cmap: str = "PuRd",
     vmin: float | None = None,
     vmax: float | None = None,
+    use_svd_number: bool = False,
```

Comment:
One could argue that the default here should be True, or to not even make it an option.
I don't see added value in adding an index option. 
The plot basically always should have started from '1' (by convention). 

On the other hand, one could argue we're catering to the more Python start from 0 convention. 

However, it does add quite a bit of complexity, in many places, so let's consult our user(s).

Reply:
We discussed it. It is easier to change it here, downstream, in the plot function with no side effects (rather than upstream in the creation of the result object). Also we discussed it's easier to change it at the level of the xarray rather than matplotlib axis manipulation.